external_parser/event_processors/joined_event.h

```diff
@@ -245,24 +259,26 @@ struct ccb_joined_event : public typed_joined_event {
       }
     }
 
-    for (auto &slot : outcomes_map) {
-      if (slot.first == -1) {
-        VW::io::logger::log_warn(
-            "CCB outcome event for event: [{}] "
-            "has slot index and slot id missing. This is an activation which "
-            "is not currently supported so it will be ignored.",
-            metadata_info.event_id);
-        continue;
-      }
+    for (size_t i = 0; i < interaction_data.size(); i++) {
+      float reward = default_reward;
```

Comment:
maybe part of the conversion PR but we can have the reward's and original reward's be member variables of `ccb_typed_event` so that we can later use them in the conversion (like [here](https://github.com/VowpalWabbit/reinforcement_learning/pull/373/files#diff-70f2ad1d28f1bb973d5cf745532f00d2c71b2aee8a7dab9370ed69521f0be095R40)). Although in the ccb case we would need a vector of rewards and a vector of original rewards